src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -108,6 +187,33 @@ private void submitNoop(final O op) {
     completeOperation(op, SUCC);
   }
 
+  private Properties createConfig(String nodeAddr) {
```

Comment:
please make such methods package-private to make the code more testable

Reply:
Done